test/hydration-test.js

```diff
@@ -0,0 +1,81 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const HydrationRepo = require('../src/Hydration');
+
+describe('HydrationRepo', () => {
+  let hydrationRepo;
+  beforeEach(() => {
+    const hydrationData = [
```

Comment:
Let's try using module.exports on the testdata set so we can refer to it here instead of having all these lines inside the test file